positions:
  - {x: -57.76, y: -53.75}
  - {x: -43.37, y: -55.96}
  - {x: -25.86, y: -53.29}
  - {x: -22.96, y: -34.24}
  - {x: -9.09, y: -34.53}
  - {x: 6.47, y: -34.80}
  - {x: 21.90, y: -34.87}
  - {x: 24.58, y: -53.52}
  - {x: 41.56, y: -56.57}
  - {x: 58.16, y: -53.99}
  - {x: 60.79, y: -38.86}
  - {x: 60.32, y: -18.99}
  - {x: 57.24, y: -3.55}
  - {x: 38.71, y: 0.25}
  - {x: 19.13, y: 0.67}
  - {x: -0.05, y: 3.97}
  - {x: -16.27, y: -0.27}
  - {x: -39.35, y: 0.29}
  - {x: -58.42, y: -3.14}
  - {x: -60.91, y: -17.16}
  - {x: -61.22, y: -37.76}

# 중앙점으로 모든 카메라 yaw 수렴
force_center_yaw: true
center_x: -0.30
center_y: -27.62

# 기본 카메라 자세
default_z: 6.0
default_pitch: -40.0
default_roll: 0.0
